examples/with-stripe-typescript/components/ClearCart.tsx

```diff
@@ -4,7 +4,10 @@ import { useShoppingCart } from 'use-shopping-cart'
 export default function ClearCart() {
   const { clearCart } = useShoppingCart()
 
-  useEffect(() => clearCart(), [clearCart])
+  useEffect(() => {
+    clearCart()
+    return () => {}
```

Comment:
You shouldn't need to return anything

Reply:
Without return it was throwing an error "destroy is not a function".